scalardb/src/scalardb/transfer.clj

```diff
@@ -97,6 +98,34 @@
          (.put tx))
     (.commit tx)))
 
+(defn- try-tx-transfer
+  [test {:keys [from to amount]}]
+  (if-let [tx (scalar/start-transaction test)]
+    (try
+      (tx-transfer tx from to amount)
+      :commit
+      (catch UnknownTransactionStatusException _
+        (swap! (:unknown-tx test) conj (.getId tx))
+        (warn "Unknown transaction: " (.getId tx))
+        :unknown-tx-status)
+      (catch Exception e
+        (warn (.getMessage e))
+        :fail))
+    :start-fail))
+
+(defn exec-transfers
+  "Execute transfers in parallel. Give the transfer function."
+  [test op transfer-fn]
+  (let [results (pmap #(transfer-fn test %) (:value op))]
+    (if (some #{:commit} results)
+      ;; return :ok when at least 1 transaction is committed
```

Comment:
`:ok` needs to be returned for this filtering even if only one transaction is committed? https://github.com/scalar-labs/scalar-jepsen/blob/3e3ad81d98d7695aa13a1b219d3c360747969f90/scalardb/src/scalardb/transfer.clj#L242

Reply:
Right, it's just for the checker.